style.css

```diff
@@ -96,20 +94,101 @@ body {
   background: #6070ff;
 }
 
+.bar .menu ::after {
+  content: "";
+  display: block;
+  z-index: 1;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background: #fff;
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background: #fff;
+}
+
+body.blur .menu::after {
+  content: "";
+  position: relative;
+  z-index: -1;
+  width: 90%;
+  height: 718px;
+  mix-blend-mode: multiply;
+}
+
+.mobile-menu {
+  display: flex;
+  flex-direction: column;
+  margin-top: 6rem;
+  width: 80%;
+  height: 184px;
+  justify-content: space-around;
+  content: normal;
+}
+
+.menu {
+  z-index: 1;
+  display: none;
+  align-content: center;
+  justify-content: space-around;
+  position: absolute;
+  top: 0;
+  list-style: none;
+  background: #6070ff;
+  -webkit-backdrop-filter: blur(15px);
+  backdrop-filter: blur(10px);
+  background-blend-mode: multiply;
+  opacity: 25;
+  width: 100%;
+  height: 718px;
+}
```

Comment:
- [ ] KIndly make sure your background is blur as the one the provided Figma design.

- [ ] KIndly make the pop-up menu cover the whole height of the visible screen in order to match the provided Figma design.

 - ### YOUR DESIGN
 
    ![image](https://user-images.githubusercontent.com/87197412/225391618-89795976-872f-4d92-a1b8-1749851e42f3.png)

 - ### FIGMA DESIGN
 
    ![image](https://user-images.githubusercontent.com/87197412/225391939-a4720686-4f82-44da-ba60-65ff8f88b780.png)